src/classes/RD2_DataMigrationBase_BATCH.cls

```diff
@@ -316,6 +318,8 @@ public abstract class RD2_DataMigrationBase_BATCH implements Database.Batchable<
         } else if (rdSettings.npe03__Open_Opportunity_Behavior__c == RD2_Constants.CloseActions.Mark_Opportunities_Closed_Lost.name()) {
             for (Opportunity opp : opps) {
                 opp.StageName = System.Label.npe03.RecurringDonationClosedLostOpportunityStage;
+                opp.Recurring_Donation_Installment_Number__c = MIGRATION_INSTALLMENT_NUMBER;
```

Comment:
Might be worth updating the method comment to explain why we're setting this field value in additional to changing the stage to ClosedLost.